jarbas/dashboard/admin/__init__.py

```diff
@@ -149,4 +157,120 @@ def get_search_results(self, request, queryset, search_term):
         return queryset, distinct
 
 
+class ReimbursementSummaryModelAdmin(PublicAdminModelAdmin):
+    change_list_template = 'dashboard/reimbursement_summary_change_list.html'
+    list_filter = (
+        list_filters.SuspiciousListFilter,
+        list_filters.HasReimbursementNumberFilter,
+        list_filters.StateListFilter,
+        list_filters.YearListFilter,
+        list_filters.MonthListFilter,
+        list_filters.DocumentTypeListFilter,
+    )
+
+    def get_period(self, request):
+        if 'year' in request.GET:
```

Comment:
This is confuse but I don't have a better suggestion 🦀
Maybe replace the `get_period` by something less context dependent. What do you think?

Reply:
Actually I think I did the opposite: I changed the name for something more context specific. But I added nice docstrings to help the reader… what do you think? It's basically c1ae375

Reply:
This is a better idea, very nice, @cuducos 🚀